lua/entities/base_wire_entity.lua

```diff
@@ -11,10 +11,11 @@ ENT.IsWire = true
 
 if CLIENT then
 	local wire_drawoutline = CreateClientConVar("wire_drawoutline", 1, true, false)
+	local beingLookedAtByLocalPlayer
 
 	function ENT:Initialize()
 		self.NextRBUpdate = CurTime() + 0.25
-		self.playerWasLookingAtMe = false
+		beingLookedAtByLocalPlayer = BaseClass.BeingLookedAtByLocalPlayer
```

Comment:
This shouldn't need to be set in the Initialize function. I wouldn't localize this anyway.

Reply:
I localized this because doing so does seem to have a performance impact, but I can take it back out if you'd prefer.

Reply:
Does setting it where the local is declared not work?

Reply:
Does it matter if it's only ever going to be called infrequently? If the entity ever gets reinitialized it'll avoid a small leak.